zerver/tests/test_message_edit.py

```diff
@@ -1342,8 +1342,8 @@ def notify(user_id: int) -> Dict[str, Any]:
             send_notification_to_new_thread=False,
             content=None,
         )
-        self.assertFalse(topic_is_muted(hamlet, stream.id, change_one_topic_name))
-        self.assertTrue(topic_is_muted(hamlet, stream.id, change_later_topic_name))
+        self.assertTrue(topic_is_muted(hamlet, stream.id, change_one_topic_name))
+        self.assertFalse(topic_is_muted(hamlet, stream.id, change_later_topic_name))
```

Comment:
This is because there's only one message in the topic, so it _does_ flip?  Skimming the test, this made me double-take until I realized the full context.